diff --git a/autoconf/bin/autoreconf.in b/autoconf/bin/autoreconf.in
index c063987..1706a52 100644
--- a/autoconf/bin/autoreconf.in
+++ b/autoconf/bin/autoreconf.in
@@ -58,7 +58,7 @@ use strict;
 $help = "Usage: $0 [OPTION] ... [DIRECTORY] ...
 
 Run `autoconf' (and `autoheader', `aclocal', `automake', `autopoint'
-(formerly `gettextize'), and `libtoolize' where appropriate)
+(formerly `gettextize'), `libtoolize', and `intltoolize' where appropriate)
 repeatedly to remake the GNU Build System files in specified
 DIRECTORIES and their subdirectories (defaulting to `.').
 
@@ -112,6 +112,7 @@ my $automake   = $ENV{'AUTOMAKE'}   || 'automake';
 my $aclocal    = $ENV{'ACLOCAL'}    || 'aclocal';
 my $libtoolize = $ENV{'LIBTOOLIZE'} || 'libtoolize';
 my $autopoint  = $ENV{'AUTOPOINT'}  || 'autopoint';
+my $intltoolize = $ENV{'INTLTOOLIZE'} || 'intltoolize';
 
 # --install -- as --add-missing in other tools.
 my $install = 0;
@@ -172,6 +173,7 @@ sub parse_args ()
       for my $prog ($autoconf, $autoheader,
 		    $automake, $aclocal,
 		    $autopoint,
+		    $intltoolize,
 		    $libtoolize)
 	{
 	  xsystem ("$prog --version | sed 1q >&2");
@@ -195,6 +197,7 @@ sub parse_args ()
       $automake   .= ' --add-missing';
       $automake   .= ' --copy' unless $symlink;
       $libtoolize .= ' --copy' unless $symlink;
+      $intltoolize .= ' --copy' unless $symlink;
     }
   # --force;
   if ($force)
@@ -207,6 +210,7 @@ sub parse_args ()
 	if $automake_supports_force_missing;
       $autopoint  .= ' --force';
       $libtoolize .= ' --force';
+      $intltoolize .= ' --force';
     }
   else
     {
@@ -230,6 +234,7 @@ sub parse_args ()
       $autoconf   .= ' --debug';
       $autoheader .= ' --debug';
       $libtoolize .= ' --debug';
+      $intltoolize .= ' --debug';
     }
   # --warnings;
   if (@warning)
@@ -446,6 +451,7 @@ sub autoreconf_current_directory ()
   my $uses_libltdl;
   my $uses_autoheader;
   my $uses_automake;
+  my $uses_intltool;
   my @subdir;
   verb "$configure_ac: tracing";
   my $traces = new Autom4te::XFile
@@ -462,6 +468,7 @@ sub autoreconf_current_directory ()
 	     'LT_CONFIG_LTDL_DIR',
 	     'AM_GNU_GETTEXT',
 	     'AM_INIT_AUTOMAKE',
+	     'IT_PROG_INTLTOOL',
 	    )
      . ' |');
   while ($_ = $traces->getline)
@@ -473,6 +480,7 @@ sub autoreconf_current_directory ()
       $uses_libltdl = 1             if /LT_CONFIG_LTDL_DIR/;
       $uses_autoheader = 1          if /AC_CONFIG_HEADERS/;
       $uses_automake = 1            if /AM_INIT_AUTOMAKE/;
+      $uses_intltool = 1            if /IT_PROG_INTLTOOL/;
       push @subdir, split (' ', $1) if /AC_CONFIG_SUBDIRS:(.*)/ && $recursive;
     }
 
@@ -537,13 +545,31 @@ sub autoreconf_current_directory ()
       verb "$configure_ac: not running libtoolize: --install not given";
     }
 
+  # -------------------- #
+  # Running intltoolize. #
+  # -------------------- #
+
+  if (!$uses_intltool)
+    {
+      verb "$configure_ac: not using Intltool";
+    }
+  elsif ($install)
+    {
+      xsystem ($intltoolize);
+      $rerun_aclocal = 1;
+    }
+  else
+    {
+      verb "$configure_ac: not running intltoolize: --install not given";
+    }
+
 
 
   # ------------------- #
   # Rerunning aclocal.  #
   # ------------------- #
 
-  # If we re-installed Libtool or Gettext, the macros might have changed.
+  # If we re-installed Libtool, Gettext or Intltool, the macros might have changed.
   # Automake also needs an up-to-date aclocal.m4.
   if ($rerun_aclocal)
     {
diff --git a/autoconf/lib/autom4te.in b/autoconf/lib/autom4te.in
index 133fb97..434c5ac 100644
--- a/autoconf/lib/autom4te.in
+++ b/autoconf/lib/autom4te.in
@@ -87,6 +87,7 @@ args: --preselect AC_PROG_LIBTOOL
 args: --preselect LT_INIT
 args: --preselect LT_CONFIG_LTDL_DIR
 args: --preselect AM_GNU_GETTEXT
+args: --preselect IT_PROG_INTLTOOL
 end-language: "Autoreconf-preselections"
 
 
