--- autoconf-2.56/lib/autoconf/libs.m4.orig	2002-10-28 09:49:59 +0300
+++ autoconf-2.56/lib/autoconf/libs.m4	2002-11-17 14:48:38 +0300
@@ -414,19 +414,6 @@
   if test "$ISC" = yes; then
     X_EXTRA_LIBS="$X_EXTRA_LIBS -lnsl_s -linet"
   else
-    # Martyn Johnson says this is needed for Ultrix, if the X
-    # libraries were built with DECnet support.  And Karl Berry says
-    # the Alpha needs dnet_stub (dnet does not exist).
-    ac_xsave_LIBS="$LIBS"; LIBS="$LIBS $X_LIBS -lX11"
-    AC_LINK_IFELSE([AC_LANG_CALL([], [XOpenDisplay])],
-                   [],
-    [AC_CHECK_LIB(dnet, dnet_ntoa, [X_EXTRA_LIBS="$X_EXTRA_LIBS -ldnet"])
-    if test $ac_cv_lib_dnet_dnet_ntoa = no; then
-      AC_CHECK_LIB(dnet_stub, dnet_ntoa,
-	[X_EXTRA_LIBS="$X_EXTRA_LIBS -ldnet_stub"])
-    fi])
-    LIBS="$ac_xsave_LIBS"
-
     # msh@cis.ufl.edu says -lnsl (and -lsocket) are needed for his 386/AT,
     # to get the SysV transport functions.
     # Chad R. Larson says the Pyramis MIS-ES running DC/OSx (SVR4)
