--- autoconf-2.59/lib/autoconf/functions.m4.vns	2003-10-27 14:13:42 +0300
+++ autoconf-2.59/lib/autoconf/functions.m4	2004-05-12 17:57:30 +0400
@@ -323,7 +323,8 @@
 AC_CACHE_CHECK([for working alloca.h], ac_cv_working_alloca_h,
 [AC_LINK_IFELSE(
        [AC_LANG_PROGRAM([[@%:@include <alloca.h>]],
-			[[char *p = (char *) alloca (2 * sizeof (int));]])],
+			[[char *p = (char *) alloca (2 * sizeof (int));
+	(void)p;]])],
 		[ac_cv_working_alloca_h=yes],
 		[ac_cv_working_alloca_h=no])])
 if test $ac_cv_working_alloca_h = yes; then
@@ -354,7 +355,8 @@
 #  endif
 # endif
 #endif
-]],                               [[char *p = (char *) alloca (1);]])],
+]],                               [[char *p = (char *) alloca (1);
+	(void)p;]])],
 		[ac_cv_func_alloca_works=yes],
 		[ac_cv_func_alloca_works=no])])
 
