 autoconf/lib/autoconf/libs.m4 |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/autoconf/lib/autoconf/libs.m4 b/autoconf/lib/autoconf/libs.m4
index 4a1301b..2938ecf 100644
--- a/autoconf/lib/autoconf/libs.m4
+++ b/autoconf/lib/autoconf/libs.m4
@@ -288,7 +288,7 @@ if test "$ac_x_libraries" = no; then
   # See if we find them without any special options.
   # Don't add to $LIBS permanently.
   ac_save_LIBS=$LIBS
-  LIBS="-lXt $LIBS"
+  LIBS="-lX11 $LIBS"
   AC_LINK_IFELSE([AC_LANG_PROGRAM([@%:@include <X11/Xlib.h>],
 				  [XrmInitialize ()])],
 		 [LIBS=$ac_save_LIBS
