diff -upk.orig autoconf-2.59.orig/bin/Makefile.am autoconf-2.59/bin/Makefile.am
--- autoconf-2.59.orig/bin/Makefile.am	2003-09-25 09:25:31 +0000
+++ autoconf-2.59/bin/Makefile.am	2006-06-20 17:15:53 +0000
@@ -39,7 +39,7 @@ edit = sed \
 	-e 's,@SHELL\@,$(SHELL),g' \
 	-e 's,@PERL\@,$(PERL),g' \
 	-e 's,@bindir\@,$(bindir),g' \
-	-e 's,@datadir\@,$(pkgdatadir),g' \
+	-e 's,@datadir\@,$(pkgdatadir)-2.5,g' \
 	-e 's,@prefix\@,$(prefix),g' \
 	-e 's,@autoconf-name\@,'`echo autoconf | sed '$(transform)'`',g' \
 	-e 's,@autoheader-name\@,'`echo autoheader | sed '$(transform)'`',g' \
diff -upk.orig autoconf-2.59.orig/bin/Makefile.in autoconf-2.59/bin/Makefile.in
--- autoconf-2.59.orig/bin/Makefile.in	2003-11-06 09:33:26 +0000
+++ autoconf-2.59/bin/Makefile.in	2006-06-20 17:15:53 +0000
@@ -189,7 +189,7 @@ edit = sed \
 	-e 's,@SHELL\@,$(SHELL),g' \
 	-e 's,@PERL\@,$(PERL),g' \
 	-e 's,@bindir\@,$(bindir),g' \
-	-e 's,@datadir\@,$(pkgdatadir),g' \
+	-e 's,@datadir\@,$(pkgdatadir)-2.5,g' \
 	-e 's,@prefix\@,$(prefix),g' \
 	-e 's,@autoconf-name\@,'`echo autoconf | sed '$(transform)'`',g' \
 	-e 's,@autoheader-name\@,'`echo autoheader | sed '$(transform)'`',g' \
diff -upk.orig autoconf-2.59.orig/lib/Makefile.am autoconf-2.59/lib/Makefile.am
--- autoconf-2.59.orig/lib/Makefile.am	2003-06-25 13:37:54 +0000
+++ autoconf-2.59/lib/Makefile.am	2006-06-20 17:15:53 +0000
@@ -25,7 +25,7 @@ edit = sed \
 	-e 's,@SHELL\@,$(SHELL),g' \
 	-e 's,@PERL\@,$(PERL),g' \
 	-e 's,@bindir\@,$(bindir),g' \
-	-e 's,@datadir\@,$(pkgdatadir),g' \
+	-e 's,@datadir\@,$(pkgdatadir)-2.5,g' \
 	-e 's,@prefix\@,$(prefix),g' \
 	-e 's,@autoconf-name\@,'`echo autoconf | sed '$(transform)'`',g' \
 	-e 's,@autoheader-name\@,'`echo autoheader | sed '$(transform)'`',g' \
diff -upk.orig autoconf-2.59.orig/lib/Makefile.in autoconf-2.59/lib/Makefile.in
--- autoconf-2.59.orig/lib/Makefile.in	2003-11-06 09:33:27 +0000
+++ autoconf-2.59/lib/Makefile.in	2006-06-20 17:15:53 +0000
@@ -126,7 +126,7 @@ edit = sed \
 	-e 's,@SHELL\@,$(SHELL),g' \
 	-e 's,@PERL\@,$(PERL),g' \
 	-e 's,@bindir\@,$(bindir),g' \
-	-e 's,@datadir\@,$(pkgdatadir),g' \
+	-e 's,@datadir\@,$(pkgdatadir)-2.5,g' \
 	-e 's,@prefix\@,$(prefix),g' \
 	-e 's,@autoconf-name\@,'`echo autoconf | sed '$(transform)'`',g' \
 	-e 's,@autoheader-name\@,'`echo autoheader | sed '$(transform)'`',g' \
