--- autoconf-2.57/lib/autoconf/headers.m4.vns2	2002-10-27 21:23:14 +0300
+++ autoconf-2.57/lib/autoconf/headers.m4	2003-08-31 23:58:49 +0400
@@ -588,6 +588,7 @@
   # /bin/cc in Irix-4.0.5 gets non-ANSI ctype macros unless using -ansi.
   AC_RUN_IFELSE([AC_LANG_SOURCE(
 [[#include <ctype.h>
+#include <stdlib.h>
 #if ((' ' & 0x0FF) == 0x020)
 # define ISLOWER(c) ('a' <= (c) && (c) <= 'z')
 # define TOUPPER(c) (ISLOWER(c) ? 'A' + ((c) - 'a') : (c))
