--- autoconf/bin/autoreconf.in
+++ autoconf/bin/autoreconf.in
@@ -58,7 +58,7 @@ use strict;
 $help = "Usage: $0 [OPTION] ... [CONFIGURE-AC or DIRECTORY] ...
 
 Run `autoconf' (and `autoheader', `aclocal', `automake', `autopoint'
-(formerly `gettextize'), and `libtoolize' where appropriate)
+(formerly `gettextize'), `libtoolize', and `intltoolize' where appropriate)
 repeatedly to remake the GNU Build System files in the DIRECTORIES or
 the directory trees driven by CONFIGURE-AC (defaulting to `.').
 
@@ -110,6 +110,7 @@ my $automake   = $ENV{'AUTOMAKE'}   || 'automake';
 my $aclocal    = $ENV{'ACLOCAL'}    || 'aclocal';
 my $libtoolize = $ENV{'LIBTOOLIZE'} || 'libtoolize';
 my $autopoint  = $ENV{'AUTOPOINT'}  || 'autopoint';
+my $intltoolize = $ENV{'INTLTOOLIZE'} || 'intltoolize';
 
 # --install -- as --add-missing in other tools.
 my $install = 0;
@@ -164,6 +165,7 @@ sub parse_args ()
       for my $prog ($autoconf, $autoheader,
 		    $automake, $aclocal,
 		    $autopoint,
+		    $intltoolize,
 		    $libtoolize)
 	{
 	  xsystem ("$prog --version | sed 1q >&2");
@@ -186,6 +188,7 @@ sub parse_args ()
       $automake   .= ' --add-missing';
       $automake   .= ' --copy' unless $symlink;
       $libtoolize .= ' --copy' unless $symlink;
+      $intltoolize .= ' --copy' unless $symlink;
     }
   # --force;
   if ($force)
@@ -197,6 +200,7 @@ sub parse_args ()
       $automake   .= ' --force-missing';
       $autopoint  .= ' --force';
       $libtoolize .= ' --force';
+      $intltoolize .= ' --force';
     }
   else
     {
@@ -215,6 +219,7 @@ sub parse_args ()
       $automake   .= ' --verbose';
       $aclocal    .= ' --verbose';
       $libtoolize .= ' --debug';
+      $intltoolize .= ' --debug';
     }
   # --warnings;
   if (@warning)
@@ -427,6 +432,7 @@ sub autoreconf_current_directory ()
   my $uses_gettext_via_traces;
   my $uses_libtool;
   my $uses_autoheader;
+  my $uses_intltool;
   my @subdir;
   verb "$configure_ac: tracing";
   my $traces = new Autom4te::XFile
@@ -440,6 +446,7 @@ sub autoreconf_current_directory ()
 	     'AC_INIT',
 	     'AC_PROG_LIBTOOL',
 	     'AM_GNU_GETTEXT',
+	     'IT_PROG_INTLTOOL',
 	    )
      . ' |');
   while ($_ = $traces->getline)
@@ -449,6 +456,7 @@ sub autoreconf_current_directory ()
       $uses_gettext_via_traces = 1  if /AM_GNU_GETTEXT/;
       $uses_libtool = 1             if /AC_PROG_LIBTOOL/;
       $uses_autoheader = 1          if /AC_CONFIG_HEADERS/;
+      $uses_intltool = 1            if /IT_PROG_INTLTOOL/;
       push @subdir, split (' ', $1) if /AC_CONFIG_SUBDIRS:(.*)/;
     }
 
@@ -509,13 +517,31 @@ sub autoreconf_current_directory ()
       verb "$configure_ac: not running libtoolize: --install not given";
     }
 
+  # -------------------- #
+  # Running intltoolize. #
+  # -------------------- #
+
+  if (!$uses_intltool)
+    {
+      verb "$configure_ac: not using Intltool";
+    }
+  elsif ($install)
+    {
+      xsystem ($intltoolize);
+      $rerun_aclocal = 1;
+    }
+  else
+    {
+      verb "$configure_ac: not running intltoolize: --install not given";
+    }
+
 
 
   # ------------------- #
   # Rerunning aclocal.  #
   # ------------------- #
 
-  # If we re-installed Libtool or Gettext, the macros might have changed.
+  # If we re-installed Libtool, Gettext or Intltool, the macros might have changed.
   # Automake also needs an up-to-date aclocal.m4.
   if ($rerun_aclocal)
     {
--- autoconf/lib/autom4te.in
+++ autoconf/lib/autom4te.in
@@ -67,6 +67,7 @@ args: --preselect AC_CONFIG_SUBDIRS
 args: --preselect AC_INIT
 args: --preselect AC_PROG_LIBTOOL
 args: --preselect AM_GNU_GETTEXT
+args: --preselect IT_PROG_INTLTOOL
 end-language: "Autoreconf-preselections"
 
 
